### PR TITLE
FMUv4pro : updates

### PR DESCRIPTION
**Pull request by klopezal**
This PR : 
- adds overcurrent sensing capability for both high-power and periphal rails
- updates mapping to V1.3 RC02 (correct pull-up/down/float configuration)
- enables the use of safety switch on V1.2 (workaround)

### CODE REVIEW

**Comment by klopezal** [comment]
@davids5 Anything more to add?

File: src/drivers/boards/px4io-v2/board_config.h [hunk 1]
```diff
@@ -80,6 +80,7 @@
 /* Safety switch button *******************************************************/
 
 #define GPIO_BTN_SAFETY (GPIO_INPUT|GPIO_CNF_INFLOAT|GPIO_MODE_INPUT|GPIO_PORTB|GPIO_PIN5)
+#define GPIO_SAFETY_PD  (GPIO_INPUT|GPIO_CNF_INPULLDWN|GPIO_MODE_INPUT|GPIO_PORTB|GPIO_PIN6)
```
Comment (davids5) [inline src/drivers/boards/px4io-v2/board_config.h]: I am not clear on this change.  On the latest print I have 
image

Are you shorting Pin 41-42 on some version of the HW? Skype?

**Comment by LorenzMeier** [comment]
@klopezal Have you seen the feedback from David?

**Comment by klopezal** [comment]
@LorenzMeier Yes, we will discuss it today on Skype and conclude.

**Comment by klopezal** [comment]
@davids5  PB6 on PIO has these options : 

pio

If possible, it would be great to have this workaround to solve the safety switch issue on V1.2 with very little overhead.

**Comment by davids5** [comment]
Understood! 

If we do this we have to dedicate the pin forever.  

Given V1.2 requires the the BYPASS_REG rework.
What is the the level of effort of adding the 1/8 watt pull down on the large pads at the connector? 

Please keep in mind, that the BYPASS_REG rework alone opens the board to vibration related failures.  Therefore it can not be reliably flown, unless [track level repair is done](http://www.circuitmedic.com/) but a certified rework house. The same would hold true for the adding the resistor. The units would be fine for bench testing. 

@LorenzMeier  - please advise.

**Comment by davids5** [comment]
@klopezal - this is a better link for [track level repair](http://www.circuitrework.com/features/772.html)

**Comment by klopezal** [comment]
There is no ground on the pads on that side of the board, so as you say adding the resistor would be uneffective unless the repair is reliable in terms of vibration level. I suggest we drop the workaround then.

**Comment by davids5** [comment]
@klopezal 

I am assuming by "workaround" you are refering the the SW one? 

If that is the case please revert 9528ae2 - FMUv4pro : workaround for safety switch and we will merge this.

**Comment by klopezal** [comment]
@davids5 Done!

**Comment by davids5** [comment]
@klopezal Thank you!

**Pull request merged**